Examples/SampleApp/iOS/Info.plist

```diff
@@ -36,5 +36,7 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string></string>
```

Comment:
Yup, it works with an empty string. A message is already provided by default along the lines of what you suggested above, see:

![screen shot 2015-04-08 at 5 09 56 pm](https://cloud.githubusercontent.com/assets/90494/7058106/236df340-de12-11e4-86f2-d596c46471fb.png)

Anything you add to `NSLocationWhenInUseUsageDescription` appears underneath that in smaller text.